src/Dotnet.Script.Core/Interactive/InteractiveRunner.cs

```diff
@@ -116,6 +116,21 @@ private async Task RunFirstScript(ScriptContext scriptContext)
                 _globals.Args.Add(arg);
 
             var compilationContext = ScriptCompiler.CreateCompilationContext<object, InteractiveScriptGlobals>(scriptContext);
+            foreach (var warning in compilationContext.Warnings)
```

Comment:
Maybe we could move this into a method in the `ScriptConsole`. Something like `DumpDiagnostics`?
We have basically the same code in the `ScriptEmitter` class